### ISSUE TITLE
hello up,i find a bug!!

### ISSUE THREAD

**Issue by ZZPHUB** [WolfChen1996/DesktopPet#1]
i am from bilibili, i find if i use my extern screen,the cat will work in the wrong way!

**Comment by WolfChen1996** [comment]
Yes, I haven't consider about multiple screens yet. We should use desktop.screenCount() to check how many screens are there. BTW why don't you use Chinese?
是的，我还没有考虑多个屏幕的问题，我们可以用 desktop.screenCount() 来看看用户有几个屏幕……此外，你什么不写中文呢？

**Comment by ZZPHUB** [comment]
thanks,i use English for being more professional and for my cet-6.hahaha.
